plugin/pxr/CMakeLists.txt

```diff
@@ -9,15 +9,34 @@ set(PXR_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX}/${INSTALL_DIR_SUFFIX})
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH}
     ${PROJECT_SOURCE_DIR}/cmake/modules
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/macros
+    ${CMAKE_CURRENT_SOURCE_DIR}/cmake/defaults
```

Comment:
Same thing here. No language standard is being specified as far as I can see.

Reply:
Not sure if I understand this but this is just a path for the module path.

The language standard are being applies by including `CXXDefaults`

See line 25 in plugin/pxr/CMakeLists.txt 

```
#------------------------------------------------------------------------------
# compiler configuration
#------------------------------------------------------------------------------
# CXXDefaults will set a variety of variables for the project.
# Consume them here. This is an effort to keep the most common
# build files readable.
include(CXXDefaults)

```

Reply:
Sorry, I worded that poorly. I meant to say this was the same thing as in `plugin/al/CMakeLists.txt` where this file wouldn't pick up the C++14 language standard and would get the old C++11 one instead. But this is also addressed by https://github.com/Autodesk/maya-usd/commit/dc898ce4b24f4b0b250d8657117937758668adde, thanks!